kaos/algorithm/view_cone.py

```diff
@@ -0,0 +1,132 @@
+""" Implementation of Viewing cone algorithm """
+from math import asin, atan, sqrt, sin, cos, pi
+
+from ai.cs import cart2sp
+from numpy import cross
+from numpy.linalg import norm
+
+from kaos.algorithm import SECONDS_PER_DAY, ANG_VEL_EARTH, THETA_NAUGHT, TimeInterval,\
+                            ViewConeFailure
+
+
+def view_cone(site_eci, sat_pos, sat_vel, q_magnitude, poi):
```

Comment:
This should have a more descriptive name that reflects why a caller might want to use it, like `reduce_poi`.